rmgpy/data/kinetics/library.py

```diff
@@ -219,6 +219,30 @@ def get_sticking_coefficient(self, T):
             
         return False
 
+    def apply_solvent_correction(self, solvent):
```

Comment:
Has all of this solvent corrections to kinetics been done on a different branch (and reviewed independently?) or is it all blurred in with the electrochemistry stuff?

Reply:
So I had a solvation correction system I built for the electrochemistry work. Yunsie was also working on solvation corrections and had her own dataset and a system that was similar, but also a bit different. So I modified my system to merge the two systems and handle both my dataset and her dataset.

Reply:
Did hers never make it into RMG then? this is the [only] chance to review both?

Reply:
Yes, this is all only here.